python/lsst/ip/diffim/dipoleMeasurement.py

```diff
@@ -310,14 +312,14 @@ def classify(self, sources):
 
             source.set(key, val)
 
-    def run(self, exposure, sources, **kwds):
+    def run(self, sources, exposure, **kwds):
```

Comment:
The parameter order was swapped.  Please update the docstring to reflect the change.